frontend/src/plugin/index.ts

```diff
@@ -244,10 +246,57 @@ export function updateSettingsPackages(
   });
 }
 
+/**
+ * Runs a plugin with the given info.
+ *
+ * This is not a closure, so it doens't have access to the variables
+ *  in the scope of the function that called it.
+ */
+function runPluginInner(info: runPluginProps) {
+  const source = info[0];
+  const packageName = info[1];
+  const packageVersion = info[2];
+  const handleError = info[3];
+  const PrivateFunction = info[4];
+  const args = info[5];
+  const values = info[6];
+  const privateRunPlugin = info[7];
```

Comment:
Same comment about expanding a const list declaration.

Reply:
This is to avoid calling the iterator, which could have the prototype overridden.